source/agora/network/RPC.d

```diff
@@ -332,6 +333,9 @@ private class RPCConnection
     ///
     private TCPConnection conn;
 
+    /// Check if peer is banned on receive
+    private RejectConnectionPredicate isBannedDg;
```

Comment:
I don't think we need this. Since this is to reject incoming connections, only point we need this is `listenRPC`